setup.cfg

```diff
@@ -7,11 +7,36 @@ all_files = 1
 upload-dir = docs/_build/html
 show-response = 1
 
+[flake8]
+max-line-length = 160
+exclude =
+    build,
+    .eggs,
+    .svn,
+    CVS,
+    .bzr,
+    .hg,
+    .git,
+    __pycache__
+select =
+    F,
+    E101,
+    E111,
+    E112,
+    E113,
+    E401,
+    E402,
+    E711,
+    E722
+
 [tool:pytest]
 minversion = 3.0
-norecursedirs = build docs/_build astropy_helpers
+norecursedirs = .eggs build docs/_build astropy_helpers
 doctest_plus = enabled
 
+[aliases]
+test=pytest
```

Comment:
Is this safe?

Reply:
`#livingdangerously`

Reply:
Ah, I see.  Because it uses the pytest-runner, it works.  Cool.  I withdraw my concern.

Reply:
The trick is you have to do:
```
pip install -e .[test]
python setup.py test
```
Without `pytest-runner` package, the error message from `python setup.py test` is a bit opaque.

Reply:
At this point, do we even need to use `setup.py test`? Isn't calling `pytest` directly less confusing?

Reply:
The current recommended best practice seems to be moving away from extra commands such as `test` and `build_sphinx` implemented in setup.py and instead moving to just running pytest at the top level.  This allows setup.py to almost become a one-liner and setup.cfg to contain all declarative setup of the package.

Anyway, happy to do this in stages.  =)